core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -115,11 +115,10 @@ export class UrlInterpolationService {
    */
   interpolateUrl(
       urlTemplate: string,
-      interpolationValues: InterpolationValuesType): string | null {
+      interpolationValues: InterpolationValuesType): string {
     if (!urlTemplate) {
       this.alertsService.fatalWarning(
         'Invalid or empty URL template passed in: \'' + urlTemplate + '\'');
-      return null;
```

Comment:
Are we expecting to stop the function execution here?

Reply:
Sorry I missed [the above comment](https://github.com/oppia/oppia/pull/13371#discussion_r667490967).

Reply:
Can you please add a code comment explaining the same. ALso, can you confirm that we have a unit test to ensure that the function execution stops after line 121?

Reply:
I updated the comment. Yes, we're expecting an error, particular unit-tests are there for each case. Actually previously when we were not hitting `return null` statements then coverage was not full
![image](https://user-images.githubusercontent.com/60519303/125405123-ffda4a80-e3d4-11eb-8d5e-939f224d1f8d.png)
 Removing them also increased the coverage